### PR TITLE
Update to 3.5.0

### PR DESCRIPTION
**Pull request by mnaoumov**

### CODE REVIEW

**Comment by mnaoumov** [comment]
@RafaelGB the permissions you gave me don't allow me to push to master directly. It says I have to go via Pull Request process and with code review and it seems you are the only reviewer who can approve

File: manifest-beta.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
 	"id": "dbfolder",
 	"name": "DB Folder",
-	"version": "3.4.0",
+	"version": "3.5.0",
```
Comment (RafaelGB) [inline manifest-beta.json:4]: I thinks it's a minor => 3.4.1

File: manifest.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
 	"id": "dbfolder",
 	"name": "DB Folder",
-	"version": "3.4.0",
+	"version": "3.5.0",
```
Comment (RafaelGB) [inline manifest.json:4]: => 3.4.1

File: docs/docs/changelog.md [hunk 3]
```diff
@@ -1,3 +1,6 @@
+## 3.5.0
```
Comment (RafaelGB) [inline docs/docs/changelog.md:1]: => 3.4.1

**Comment by RafaelGB** [comment]
the releases yaml was updated to use the tag

**Comment by RafaelGB** [comment]
Sure, we can maintain the 3.5.0
The criterion I was following was to raise the Z in case of only fixing bugs in a version

**Review by RafaelGB — APPROVED** [review]

**Comment by RafaelGB** [comment]
https://github.com/RafaelGB/obsidian-db-folder/releases/tag/3.5.0

**Comment by RafaelGB** [comment]
Once I published https://github.com/RafaelGB/Obsidian-ZettlelFlow maybe I can develop some things here. Again, thanks for your work!

**Pull request merged**